app/javascript/shared/helpers/string.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Shall we move this @chatwoot/utils?

Reply:
Done here: https://github.com/chatwoot/utils/pull/18